testcases/extensions-map.json

```diff
@@ -0,0 +1,18 @@
+{
+    "specversion": "0.4-wip",
+    "id": "123",
+
+    "source": "https://cloudevents.io/extensions-map",
+    "type": "io.cloudevents.extensions-map-test",
+
+    /***
+     * EXTENSIONS, with a map value
+     */
+    "this0is0ok": {
+        "\": "ok",
```

Comment:
I don't think `\` is a valid char in a name unless it's part of an escape sequence.

Reply:
I think perhaps you forgot the 2nd `\` ?

Reply:
I forgot the second `"`.

Reply:
I forget the second `"`, to make this `"\""`